plugin-procfs/README.md

```diff
@@ -1,3 +1,138 @@
+|4|`PTraceable` : Procfs should only contain `/proc/<pid>/` directories that the caller can ptrace|
+
+## Metrics
+
+There are various information collected by this plugin relative to Kernel, CPU, memory and processes :
+
+|Name|Type|Unit|Description|Ressource|RessourceConsumer|Attributes|
+|----|----|----|-----------|---------|-----------------|----------|
+|`kernel_cpu_time`|CounterDiff|MilliSecond|Time during the CPU is busy|LocalMachine|LocalMachine|[cpu_state](#cpu_state)|
+|`kernel_context_switches`|CounterDiff|None|Number of context switches : Operation allowing a single CPU to manage multiple processes efficiently, involves saving the state of a currently running process and loading the state of another process, enabling multitasking and optimal CPU utilization|LocalMachine|LocalMachine||
+|`kernel_new_forks`|CounterDiff|None|Number of forked operations : When a process creates a copy of itself|LocalMachine|LocalMachine||
+|`kernel_n_procs_running`|Gauge|None|Number of processes in a runnable state|LocalMachine|LocalMachine||
+|`kernel_n_procs_blocked`|Gauge|None|Numbers of processes that are blocked on input/output operations|LocalMachine|LocalMachine||
+|`cpu_time_delta`|CounterDiff|NanoSecond|CPU usage|LocalMachine|Process|[kind](#kind)|
+|`memory_usage`|Gauge|Bytes|Memory usage|LocalMachine|Process|[kind](#kind)|
+
+And in case we configure the plugin with the collect of detailed memory metrics :
+
+|Name|Type|Unit|Description|Ressource|RessourceConsumer|Attributes|
+|----|----|----|-----------|---------|-----------------|----------|
+|`mem_total`|Gauge|Bytes|Total amount of physical memory available in the system|LocalMachine|LocalMachine||
+|`mem_free`|Gauge|Bytes|Refers to truly free memory, which is not used by processes, caches or buffers|LocalMachine|LocalMachine||
+|`mem_available`|Gauge|Bytes|More accurate estimate of the actual available memory, taking into account free memory and also caches and buffers that can be freed|LocalMachine|LocalMachine||
+|`cached`|Gauge|Bytes|Cached is the memory used for ROM cache (page cache) in memory|LocalMachine|LocalMachine||
+|`swap_cached`|Gauge|Bytes|Memory that was placed in swap and brought back to RAM, but remains in swap file|LocalMachine|LocalMachine||
+|`active`|Gauge|Bytes|Memory that has been used recently|LocalMachine|LocalMachine||
+|`inactive`|Gauge|Bytes|Used memory that can be reclaimed, freed, or moved to swap.|LocalMachine|LocalMachine||
+|`mapped`|Gauge|Bytes|Memory used for files associated with processes in memory, file and memory mapping.|LocalMachine|LocalMachine||
+
+### Attributes
+
+#### Kind
+
+The kind of the memory is the allocated memory space reserved by the system or the hardware :
+
+|Value|Description|
+|-----|-----------|
+|`resident`|Physical memory actually used by a process|
+|`shared`|Part of memory used simultaneously by several processes|
+|`virtual`|Total virtual memory address space requested by a process|
```

Comment:
The descriptions here are a bit simplistic, it's more complicated than that. IMO, it would be better to stay factual and use the same description as the manual. See the comments in the `procfs-core` crate, structure `StatM`, and [`man proc/pid/statm`](https://man7.org/linux/man-pages/man5/proc_pid_statm.5.html) and [`man /proc/pid/status`](https://man7.org/linux/man-pages/man5/proc_pid_status.5.html).